exes/ethercat/CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ if (BUILD_TESTING)
   add_subdirectory(tests)
 endif ()
 
+configure_file("${CMAKE_CURRENT_LIST_DIR}/systemd/ethercat@.service" "${CMAKE_BINARY_DIR}/systemd/ethercat@.service")
+
```

Comment:
Does the install in line 50 need to point to binary for?

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_BINARY_DIR.html#variable:CMAKE_BINARY_DIR

this variable points to the top level in the build directory, do you have any other suggestion?

Reply:
I cannot place a comment on line 50 on github, the reason I mentioned it directly. The install is broken because of this configure_file, and needs to be fixed there. Since you now place it in binary dir you need to install from binary dir.
 If you test the installer, you will see the service file is still the same:
![image](https://github.com/Skaginn3x/framework/assets/15651464/79010f34-ec2a-45ea-82da-f094ce510da3)

Reply:
Changed the path in the installer, this should work now.